chia/_tests/core/full_node/test_full_node.py

```diff
@@ -2504,9 +2504,9 @@ async def validate_coin_set(coin_store: CoinStore, blocks: list[FullBlock]) -> N
             assert records == {}
             continue
 
-        if len(block.transactions_generator_ref_list) > 0:  # pragma: no cover
-            # TODO: Support block references
-            assert False
+        # TODO: Support block references
+        # if len(block.transactions_generator_ref_list) > 0:
+        #    assert False
```

Comment:
This used to work despite lacking block references support. Are we hitting this assert now despite still lacking support? can we expand the TODO comment to provide more context on the commented out code?

Reply:
this assert used to not trigger because the "dummy block references" feature was broken, and the test chains did not have block references. Now that I'm fixing the dummy block references, this assert would fail. The reason it's OK to ignore the block references is that they are still just "dummy". i.e. they're not used by the block generator, so we still don't need to support them.